.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,78 @@
+name: Continuous Integration Pipeline
+
+on:
+  push:
+    branches:
+      - '**'
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'LICENSE'
+  pull_request:
+    branches:
+      - '**'
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'LICENSE'
```

Comment:
This is what I added - AFAIK you don't need separate definition for pull_request because it's covered under push

```
on:
  push:
    paths-ignore:
      - '**.md'
      - '**.txt'
      - '**.json'
      - '**.yml'
      - '**.properties'
      - '**.xml'
      - '**.sql'
      - '**.csv'
      - '**.zip'
      - 'docs/**'
      - '.gitignore'
```

Reply:
It makes sense, let me resolve that